datahub-frontend/conf/application.conf

```diff
@@ -191,6 +191,8 @@ analytics.enabled = ${?DATAHUB_ANALYTICS_ENABLED}
 analytics.kafka.bootstrap.server = ${KAFKA_BOOTSTRAP_SERVER}
 analytics.tracking.topic = DataHubUsageEvent_v1
 analytics.tracking.topic = ${?DATAHUB_TRACKING_TOPIC}
+analytics.kafka.delivery.timeout.ms = 30000
```

Comment:
What was the default?

Reply:
15 minutes is the default, this is now in line with GMS's 30 seconds.